instrumentation/opentelemetry-api/opentelemetry-api-1.10/javaagent/build.gradle.kts

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 dependencies {
-  compileOnly(project(":opentelemetry-api-shaded-for-instrumenting", configuration = "shadow"))
+  compileOnly(project(":opentelemetry-api-shaded-for-instrumenting", configuration = "v1_10"))
```

Comment:
IntelliJ kept adding back the `@Override` annotation no matter how I configured it when this module was compiled against the latest version - I had to add some gradle voodoo to generate shaded 1.10 metrics api.

Reply:
wow, nice